frontend/.env.example

```
@@ -1,5 +1,7 @@
-# Frontend Environment Variables
-NEXT_PUBLIC_BACKEND_URL=http://localhost:3000
+# # Frontend Environment Variables
+# NEXT_PUBLIC_BACKEND_URL=http://localhost:3000
 
-# Optional Metadata
-NEXT_PUBLIC_APP_URL=http://localhost:3001
+# # Optional Metadata
+# NEXT_PUBLIC_APP_URL=http://localhost:3001
+NEXT_PUBLIC_BACKEND_URL=http://localhost:3000
+SRA_API_KEY=sra_live_966ea176018b1060ebda9a810d2e5fd358c2e6513ad0d5b56a026923a269b512
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)